src/Features/VisualBasic/Portable/VBFeaturesResources.Designer.vb

```diff
@@ -1731,6 +1749,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.VBFeaturesResources
         End Property
         
         '''<summary>
+        '''  Looks up a localized string similar to (Unknown).
+        '''</summary>
+        Friend ReadOnly Property paren_unknown_paren() As String
+            Get
+                Return ResourceManager.GetString("paren_unknown_paren", resourceCulture)
```

Comment:
This is the same as the C# one.  We should put the resoburce in a common location and reference the same resource from both language.